tsfc/driver.py

```diff
@@ -361,6 +362,12 @@ def compile_expression_dual_evaluation(expression, to_element, coordinates, inte
         point_set = PointSet(qpoints)
         config = kernel_cfg.copy()
         config.update(point_set=point_set)
+
+        # Fix for interpolating QuadratureWeight into a QuadratureElement
```

Comment:
I think this comment only makes sense now, and not later. Perhaps "Allow interpolation onto QuadratureElements to refer to the quadrature rule they represent"?

Reply:
Fixed, thanks.